nixos/tests/microsoft-identity-broker.nix

```diff
@@ -0,0 +1,16 @@
+import ./make-test-python.nix ({ pkgs, ...} : {
+  name = "microsoft-identity-broker";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ rhysmdnz ];
+  };
+
+  nodes.machine =
+    { pkgs, ... }:
+    { services.intune.enable=true;
+    };
+
+  testScript = ''
+    start_all()
+    machine.succeed("systemctl start microsoft-identity-device-broker.service")
```

Comment:
We should test more than just that the package starts

Reply:
I would love to, but how it works isn't defined anywhere. And the user facing interface requires a network and a Microsoft account.

We could make tests based on reversing how it works, but that doesn't feel like a sustainable use of time.

This catches the real obvious we broke the Java

Reply:
I've expanded the test to cover more of the intune binaries